services/algolia-sync/src/components/algolia/site-sync.js

```diff
@@ -0,0 +1,81 @@
+const gql = require('graphql-tag');
+const algoliasearch = require('algoliasearch');
+const cliProgress = require('cli-progress');
+const apollo = require('../graphql/query');
+const helpers = require('./helpers');
+require('dotenv').config();
+
+const client = algoliasearch(process.env.ALGOLIA_APPID, process.env.ALGOLIA_API_KEY);
+
+const bar = new cliProgress.SingleBar({}, cliProgress.Presets.shades_classic);
+
+const query = () => (gql`
```

Comment:
No need to return a function here as you're not executing with arguments.
```suggestion
const query = gql`
```

Reply:
No clue how it got like that, must have been the auto lint format. Anyways, it should be fixed now.